web/templates/stream_settings/selected_stream_title.hbs

```diff
@@ -1,5 +1,15 @@
+{{#if preview_url}}
+<a href="{{preview_url}}" class="tippy-zulip-tooltip" data-tooltip-template-id="view-stream-tooltip-template" data-tippy-placement="bottom">
+    {{> stream_privacy_icon
+      invite_only=sub.invite_only
+      is_web_public=sub.is_web_public
+      color=title_icon_color }}
+    <span class="stream-name-title">{{sub.name}}</span>
+</a>
+{{else}}
 {{> stream_privacy_icon
   invite_only=sub.invite_only
   is_web_public=sub.is_web_public
   color=title_icon_color }}
 <span class="stream-name-title">{{sub.name}}</span>
+{{/if}}
```

Comment:
I coulld not think of anything to avoid code duplicacy here. I tried something like 
```
{{# if preview_url}}
<a href=.....>
{{/if}}
.........
{{# if preview_url}}
</a>
{{/if}}
```   
But it does not passed lint test.